recipes/augustus/3.2.2/meta.yaml

```diff
@@ -43,3 +43,69 @@ source:
 test:
   commands:
     - augustus --version
+    - autoAug.pl --help &>/dev/null
+    - bedgraph2wig.pl --help &>/dev/null
+    - blat2gbrowse.pl --help 2>&1 | grep 'convert blat file to gbrowse file' &>/dev/null
+    - blat2hints.pl --help 
+    - block2prfl.pl --help 2>&1 | grep "default.qij"
+    - cegma2gff.pl --help 2>&1 | grep "Could not open CEGMA file --help"
+    - checkParamArchive.pl --help 2>&1 | grep "checkParamArchive.pl param-archive.tar.gz projectDir > utrPossible 2> errors"
+    - cleanDOSfasta.pl --help 2>&1 | grep "Could not open file --help"
```

Comment:
Its suspicious software. The vast majority of the scripts included don't exit 0 with --help, even if they have usage. Most will at least spit out a usage docs, and none fail because they are missing perl packages.

I have a couple researchers who do technology watch, and they know the dangers, but software like this should come with a warning label.